src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -203,7 +203,7 @@ function updatePhaseTableChart(flux) {
             x: {columnOrExpr: LC.PHASE_CNAME, options: 'grid'},
             y: {columnOrExpr: flux, options: 'grid,flip'}
         };
-        loadXYPlot({chartId: LC.PHASE_FOLDED, tblId: LC.PHASE_FOLDED, xyPlotParams});
+        loadXYPlot({chartId: LC.PHASE_FOLDED, tblId: LC.PHASE_FOLDED, xyPlotParams, help_id: 'main1TSV.plot'});
```

Comment:
Instead of passing the same help_id into the individual chart, it should be passed into ChartContainer.
Although in this case, it may appear the same.